### PR TITLE
Feature #1

### PR DESCRIPTION
**Pull request by mayuir2014**
Feature Added - Variable number of items in row.

### CODE REVIEW

File: properties.schema [hunk 1]
```diff
@@ -65,6 +65,14 @@
           }
         }
       }
+    },
+    "_inRow": {
+      "type":"number",
+      "required":true,
+      "default": 2,
+      "inputType": "Number",
```
Comment (dancgray) [inline properties.schema]: Needs a title:

"title": "Number in Row",

**Comment by dancgray** [comment]
Looks good, have not tested.

Just to be picky, can the checkInRow function be indented the same as the rest of the file. Sorry :)

**Comment by dancgray** [comment]
What happens if you have 3 flip cards but 2 in a row?

File: properties.schema (hunk 1 above)
Comment (mayuir2014) [inline properties.schema]: Done.

**Comment by dancgray** [comment]
+1, cool

**Pull request merged**